cluster/manifests/kube-node-ready-controller/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kube-node-ready-controller
+  namespace: kube-system
+  labels:
+    application: kube-node-ready-controller
+    version: latest
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      application: kube-node-ready-controller
+  template:
+    metadata:
+      labels:
+        application: kube-node-ready-controller
+        version: latest
+    spec:
+      tolerations:
+      - key: node.alpha.kubernetes.io/notReady-workload
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: kube-node-ready-controller
+        image: mikkeloscar/kube-node-ready-controller:latest
+        args:
+        - --not-ready-taint-name=node.alpha.kubernetes.io/notReady-workload
```

Comment:
Do we want a better name here?

Reply:
suggestions welcome :)

Reply:
`lifecycle-status=initializing:NoSchedule` :D

Reply:
Or `node-status=initializing:NoSchedule`

Reply:
I think something with `notReady` is better than initializing because it could potentially be removed after some time, so not only during initializing of the node.

Reply:
Good point! My concerns with the current name are that it includes `kubernetes.io` which I think we shouldn't use and the mixed casing (`notReady-workload`). I don't have better suggestions than something like `node-not-ready`, though. :(

Reply:
That's fair. I changed it to `node-not-ready` for now.

@linki WDYT?

Reply:
`node-not-ready` is fine for me.